src/renderer/families/ethereum/operationDetails.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from "react";
+import { Trans } from "react-i18next";
+import type { AccountLike } from "@ledgerhq/live-common/lib/types";
+
+import Box from "~/renderer/components/Box";
+import { OpDetailsTitle, OpDetailsData } from "~/renderer/modals/OperationDetails/styledComponents";
+
+type OperationDetailsExtraProps = {
+  extra: { [key: string]: string },
+  type: string,
+  account: ?AccountLike,
+};
+
+const OperationDetailsExtra = ({ extra, type }: OperationDetailsExtraProps) => {
+  // $FlowFixMe somehting wrong with .map
```

Comment:
you could use lodash's way to do entries. Object.entries <> FlowType support have issue.

Reply:
using lodash entries, it does not solve the flow problem